runtime/hookable/hookableshared.mk.ftl

```diff
@@ -1,5 +1,5 @@
 ###############################################################################
-# Copyright (c) 2016, 2017 IBM Corp. and others
+# Copyright (c) 2016, 2020 IBM Corp. and others
```

Comment:
This file isn't changing, so the copyright date need not change.

Reply:
Removed.